### DISCUSSION TITLE
Clear Filter on change of Filterable

### DISCUSSION THREAD

**Discussion by bastigerner** [Megabit/Blazorise#1828] [Q&A]
Hey,

a quick question. I am using the FilterTemplate with autocomplete inside a datagrid.
I also have a button to change Filterable inside the DataGrid.

My question is: is it possible to clear my filter when I set Filterable to false?

Because when I hide the filters it still shows me the current filtered list.

**Comment by stsrki** [comment]
Not sure but I believe this is fixed with #1719

It will be available once 0.9.3 is released.

**Reply by bastigerner** [reply]
Okay good to hear!
Is there any date for the release known yet?

**Reply by stsrki** [reply]
No date. Basically when it's done. (working alone 😅)

You can track it on https://github.com/stsrki/Blazorise/milestone/29
